README.md

```diff
@@ -1,57 +1,113 @@
+
+# How the project is structured
+
+The following table describes all files within this solution:
+
+| Folder         | FileName                    | Description                                                                                                                                                                                                                                                               |
+|----------------|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Root\\^           | main.java                   | Reads configuration, authenticates, executes all operations
+| Root\\^           | Creation.java               | Class that contains all resource creation loops, following the hierarchy logic in order to successfully deploy Azure NetApp Files resources
+| Root\\^           | Cleanup.java                | Class that performs cleanup of all artifacts that were created during this sample application. Its call is commented out by default in main.java
+| Root\\^\common    | CommonSdk.java              | Class dedicated to common operations related to ANF's SDK
+| Root\\^\common    | ResourceUriUtils.java       | Class that exposes a few methods that help parsing Uri's, building new Uri's, or getting a resource name from a Uri, etc
+| Root\\^\common    | ServiceCredentialsAuth.java | A small support class for extracting and creating credentials from a File
+| Root\\^\common    | Utils.java                  | Class that exposes methods that help with getting the configuration object, byte conversion, etc
+| Root\\^\model     | AzureAuthInfo.java          | POJO class to hold authentication information
+>\\^ == src/main/java/sdk/sample                                                               |
+
+# How to run the console application
+
+1. Clone it locally
+    ```powershell
+    git clone https://github.com/Azure-Samples/netappfiles-java-nfs4.1-sdk-sample.git
+    ```
+1. Make sure you change the variables located at **.netappfiles-java-crr-sdk-sample\src\main\main.java at RunAsync method.**
+1. Change folder to **.\anf-java-crr-sdk-sample**
+1. Since we're using service principal authentication flow, make sure you have the **azureauth.json** and its environment variable with the path to it defined (as previously described)
+1. Compile the console application
+    ```powershell
+    mvn clean compile
+    ```
+1. Run the console application
+    ```powershell
+    mvn exec:java -Dexec.mainClass="sdk.sample.main"
+
+Sample output
+![e2e execution](./media/e2e-execution.png)
+
+# References
+
```

Comment:
I think we have documentation available for CRR, can you please reference that here?